source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
We don't have to use `.nav-anchor` as part of the selector here since `id` should be unique anyway. Plus as long as a DOM element has `id` we can create anchor for it (it doesn't have to have any specific classname)